.gitignore

```diff
@@ -55,3 +55,12 @@ public/sitemap*
 # Ignore local configuration
 .env.development
 .env.test
+
+# Ignore developer cruft
+**/.DS_Store
+.sass-cache
+
+# Ignore branding adjustments, user should track that separately
+# (ref app/assets/stylesheets/branding/README.md).
+app/assets/stylesheets/branding
+app/assets/images/branding
```

Comment:
After reading the link, I think `app` makes sense.  I view these files essentially as configuration files which live in the same folder as the app, the same as a _config.yml would, for example.  The comments given for `lib/assets` and `vendor/assets` don't feel applicable.